queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
-
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals;
+SELECT * from animals where name like '%mon';
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' and '2019-01-01';
+SELECT name from animals where escape_attempts <3;
```

Comment:
- [ ] For this one, `list the name of all animals that are neutered and have less than 3 escape attempts.` You are missing adding conditions for the `neutered` case. kindly add this. Your code might look like this:

```sql
SELECT name FROM animals WHERE neutered = 'true' AND escape_attempts < '3';
```